contrib/unbound-cert-setup.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# Generally based on /usr/sbin/unbound-control-setup but adapted to catch
+# up to ~2010. You know, x509v3, secp384r1, AKIs, stuff like that.
+
+# directory for files
+DESTDIR=/etc/unbound
+
+# validity period for certificates
+DAYS=7200
```

Comment:
I know this isn't publicly trusted, but I'd make this 397 anyway. Make sure we rotate things frequently.